src/main/scala/extractor/Main.scala

```diff
@@ -12,7 +12,7 @@ object Main {
       case Some(conf) => {
         println(s"Root: ${conf.root}, Multiple projects: ${conf.multidir}")
```

Comment:
sounds like a debugging stuff - for that you can use the Logging trait (cf. SemanticsdbTest)

Reply:
Another style is to write:

```scala
object Main extends App {
  private def run(conf: Config) {
         val walker = new SingleProjectWalker(conf.root)
         val results = ImplicitParamsToCSV(walker)
         val results = walker.run(ExtractImplicits)
         CSV.dumpFiles(conf.root, results)
  }

 Cli(args) map run
}
```